twap/go.mod

```diff
@@ -0,0 +1,11 @@
+module github.com/blocky/as-demo
+
+go 1.22
+
+require github.com/stretchr/testify v1.10.0
+
+require (
+	github.com/davecgh/go-spew v1.1.1 // indirect
+	github.com/pmezard/go-difflib v1.0.0 // indirect
+	gopkg.in/yaml.v3 v3.0.1 // indirect
+)
```

Comment:
I added these dependencies to make testing of the TWAP function easier. I verified through that adding them does not increase the size of `x.wasm`